cmd/prometheus/main.go

```diff
@@ -1005,3 +1016,44 @@ func (opts tsdbOptions) ToTSDBOptions() tsdb.Options {
 		MaxBlockDuration:       int64(time.Duration(opts.MaxBlockDuration) / time.Millisecond),
 	}
 }
+
+func initTracing(logger log.Logger) (io.Closer, error) {
+	// Set tracing configuration defaults.
+	cfg := &jcfg.Configuration{
+		ServiceName: "prometheus",
+		Disabled:    true,
+	}
+
+	// Available options can be seen here:
+	// https://github.com/jaegertracing/jaeger-client-go/blob/master/config/config_env.go
+	cfg, err := cfg.FromEnv()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get tracing config from environment")
```

Comment:
https://github.com/prometheus/prometheus/issues/6972#issuecomment-598137385 -> please capitalize

Reply:
no, this an err not a message. sorry